frontend/src/app/register/page.tsx

```
@@ -11,105 +11,113 @@ const kelly = Kelly_Slab({
+                    type="text"
+                    placeholder="username"
                     className="input secondary-theme dark-font w-full"
                     required
                   />
                 </div>
-                <div className="">
+                <div>
                   <label className="label">
-                    <span className="label-text text-xs md:text-sm">Confirm Password</span>
+                    <span className="label-text text-xs md:text-sm">Email</span>
                   </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)